packages/now-next/src/utils.ts

```diff
@@ -308,6 +308,7 @@ type RoutesManifestRegex = {
 };
 
 export type RoutesManifest = {
+  basePath: string;
```

Comment:
This can be undefined on older versions of next.

```suggestion
  basePath: string | undefined;
```

Reply:
Updated